tests/general/InvalidLinkBearTest.py

```diff
@@ -203,3 +203,29 @@ def test_pip_vcs_url(self):
 
         for line in brokenlink_at_hash.splitlines():
             self.assertResult(invalid_file=[line])
+
+    def test_links_to_ignore(self):
+        valid_file = """http://httpbin.org/status/200
+        http://httpbin.org/status/201
+        http://coalaisthebest.com/
+        http://httpbin.org/status/404
+        http://httpbin.org/status/410
+        http://httpbin.org/status/500
+        http://httpbin.org/status/503
+        http://www.notexample.com/404
+        http://exampe.com/404
+        http://example.co.in/404""".splitlines()
+
+        link_ignore_list = [
+                           'http://coalaisthebest.com/',
+                           'http://httpbin.org/status/404',
+                           'http://httpbin.org/status/410',
+                           'http://httpbin.org/status/500',
+                           'http://httpbin.org/status/503',
```

Comment:
Can replace with a glob... to test if the **globbing** is working.
`http://httpbin.org/status/4[0-9][0-9]`
Similarly for `50x`..

Other then that **LGTM** :+1: